Formula/judy.rb

```diff
@@ -0,0 +1,48 @@
+class Judy < Formula
+  desc "State-of-the-art C library that implements a sparse dynamic array"
+  homepage "https://judy.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/judy/judy/Judy-1.0.5/Judy-1.0.5.tar.gz"
+  sha256 "d2704089f85fdb6f2cd7e77be21170ced4b4375c03ef1ad4cf1075bd414a63eb"
+  license "LGPL-2.1-or-later"
+
+  def install
+    system "./configure", *std_configure_args, "--disable-shared"
```

Comment:
```suggestion
    system "./configure", *std_configure_args
```
Since this is a separate formula now we should use shared libraries.